sdk/storage/Azure.Storage.Files.DataLake/tests/DirectoryClientTests.cs

```diff
@@ -1015,6 +1017,1233 @@ public async Task SetAccessControlAsync_ConditionsFail()
             }
         }
 
+        [Test]
+        [ServiceVersion(Min = DataLakeClientOptions.ServiceVersion.V2019_12_12)]
```

Comment:
Will Set Recursive ACL be released under the 2019-12-12 service version?  Or 2020-02-10?

Reply:
When it's released I think everything except force flag should be working with 12-12 (bits are there). Tests that test force flag should be marked with 02-10